wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java

```diff
@@ -177,19 +182,24 @@ private static Path convertOnnx(String modelId, String optimization) throws IOEx
             throw new IllegalArgumentException("Unsupported optimization level: " + optimization);
         }
 
-        String[] cmd = {
-            "djl-convert",
-            "--output-dir",
-            repoDir.toAbsolutePath().toString(),
-            "--output-format",
-            "OnnxRuntime",
-            "-m",
-            modelId,
-            "--optimize",
-            optimization,
-            "--device",
-            hasCuda ? "cuda" : "cpu"
-        };
+        List<String> cmd =
+                Arrays.asList(
```

Comment:
This is a readonly list, can not append more

Reply:
Done